build.gradle

```diff
@@ -16,10 +16,11 @@ buildscript {
         mavenLocal() // To check for locally published trace-gradle-plugin artifact
         mavenCentral()
         google()
+        maven { url "https://oss.sonatype.org/content/repositories/snapshots" }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.1.3'
-        classpath "com.vanniktech:gradle-android-junit-jacoco-plugin:0.16.0"
+        classpath 'com.android.tools.build:gradle:4.2.0'
+        classpath "com.vanniktech:gradle-android-junit-jacoco-plugin:0.17.0-SNAPSHOT"
```

Comment:
Is there a reason for using a snapshot dependency? Generally I would try to avoid, as it is under constant development, and we could experience failing builds.

Reply:
the snapshot dependency lets us restrict the reports to just the xml or html - which in the end we're not using - i'll move it back and see if it affects the reports.